image_width: 1280
image_height: 720
camera_name: csi_cam_0
camera_matrix:
  rows: 3
  cols: 3
  data: [ 8.6640902038184322e+02,    0.     ,  6.3950000000000000e+02,
            0.     ,  8.6640902038184322e+02,  3.5950000000000000e+02,
            0.     ,    0.     ,    1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [4.0395817397259302e-03, -2.3106248331038973e-01, 0.0, 0.0, 1.3816060671045854e-01]
rectification_matrix:
  rows: 3
  cols: 3
  data: [ 1.,  0.,  0.,
          0.,  1.,  0.,
          0.,  0.,  1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [ 362.20416,    0.     ,  256.10214,    0.     ,
            0.     ,  405.09357,  123.67604,    0.     ,
            0.     ,    0.     ,    1.     ,    0.     ]